tests/e2e-playwright/Makefile

```diff
@@ -23,6 +23,37 @@ requirements: ## compiles pip requirements (.in -> .txt)
 install-dev install-prod install-ci: _check_venv_active ## install app in development/production or CI mode
 	# installing in $(subst install-,,$@) mode
 	pip-sync requirements/$(subst install-,,$@).txt
+	# installing playwright dependencies
+	@playwright install
+
+
+get_my_ip := $(shell hostname --all-ip-addresses | cut --delimiter=" " --fields=1)
+
+.PHONY: test-sleepers
+test-sleepers: _check_venv_active ## runs sleepers test on local deploy
+	# Note that you need a user
+	@pytest \
+		-sxvv \
+		--color=yes \
+		--pdb \
+		--product-url=http://$(get_my_ip):9081 \
+		--product-billable=false \
+		--autoregister \
+		$(CURDIR)/tests/sleepers.py
+
+
+.PHONY: test-sleepers-dev
+test-sleepers-dev: _check_venv_active ## runs sleepers test on local deploy
```

Comment:
Q: what is the difference between these two commands?

Reply:
```--headed```
means that you see the browser while you run the test